CONTRIBUTING.md

```diff
@@ -1,19 +1,19 @@
-# OpenSDS
+# Soda
 
-[![Go Report Card](https://goreportcard.com/badge/github.com/opensds/opensds?branch=master)](https://goreportcard.com/report/github.com/opensds/opensds)
-[![Build Status](https://travis-ci.org/opensds/opensds.svg?branch=master)](https://travis-ci.org/opensds/opensds)
-[![Coverage Status](https://coveralls.io/repos/github/opensds/opensds/badge.svg?branch=master)](https://coveralls.io/github/opensds/opensds?branch=master)
+[![Go Report Card](https://goreportcard.com/badge/github.com/soda/soda?branch=master)](https://goreportcard.com/report/github.com/soda/soda)
+[![Build Status](https://travis-ci.org/soda/soda.svg?branch=master)](https://travis-ci.org/soda/soda)
+[![Coverage Status](https://coveralls.io/repos/github/soda/soda/badge.svg?branch=master)](https://coveralls.io/github/soda/soda?branch=master)
 
-<img src="https://www.opensds.io/wp-content/uploads/sites/18/2016/11/logo_opensds.png" width="100">
+<img src="https://www.soda.io/wp-content/uploads/sites/18/2016/11/logo_soda.png" width="100">
```

Comment:
This will not work. The SODA website is incorrect. Additionally the image link may not work even if the website is correct as the image may not be there.